__tests__/feeds.ts

```diff
@@ -1142,6 +1142,30 @@ describe('query sourceFeed', () => {
       'FORBIDDEN',
     );
   });
+
+  describe('vordr', () => {
+    it('should filter out posts that vordr has prevented', async () => {
+      loggedUser = '1';
+      await con
+        .getRepository(ArticlePost)
+        .update({ id: 'p5' }, { authorId: '2', flags: { vordr: true } });
```

Comment:
I think this potentially will introduce flaky tests as with commentFeed where due to parallel updates from tests sometimes vordr will get triggered in other tests. So maybe solution is to use different ids/fixtures for vordr test. Did not manage to look at that failing one yet.

Reply:
I added specific post for those tests